example/src/main/java/com/quickbirdstudios/example/ui/main/MainActivity.kt

```diff
@@ -21,6 +21,7 @@ import com.quickbirdstudios.surveykit.NavigationRule
 import com.quickbirdstudios.surveykit.StepIdentifier
 import com.quickbirdstudios.surveykit.SurveyTheme
 import com.quickbirdstudios.surveykit.TextChoice
+import com.quickbirdstudios.surveykit.backend.address.YandexAddressSuggestionProvider
```

Comment:
Hello thanks for your major effort and the pull request. 
Why are you using Yandex exactly?

Reply:
I forgot this class to move example app. I'm expecting like that library users can implement their address suggestion implementation. Yandex is just an example.

Reply:
I moved to yandex provider to example app. I don't want to remove it. Someone can be need see custom implementation. And also I implement a new address provider `GeocoderAddressSuggestionProvider` based android geocoder.